doc/template.html

```diff
@@ -45,5 +45,6 @@
   <script src="assets/sh_main.js"></script>
   <script src="assets/sh_javascript.min.js"></script>
   <script>highlight(undefined, undefined, 'pre');</script>
+  __TRACKING__
```

Comment:
yeah was just suggesting we could avoid it altogether (replacing with empty) by making it a html comment.